lib/core/src/animation/physics/inertia.rs

```diff
@@ -280,10 +314,12 @@ pub struct PhysicsSimulator {
 
 impl PhysicsSimulator {
     /// Simulates `Properties` and inputs `Kinematics`' position in `PhysicsCallback`.
-    pub fn new<F:PhysicsCallback>
-    ( steps_per_second:f64
-    , mut properties:PhysicsProperties
-    , mut callback:F) -> Self {
+    pub fn new<F:PhysicsCallback,Properties:Into<PhysicsProperties>>
+    ( steps_per_second : f64
+    , properties       : Properties
+    , thresholds       : SimulationThresholds
```

Comment:
Why don't we default the thresholds to some sensible, default values? Constructors should be minimal. If the user wants to change the threshold, they can do it after creating the `PhysicsSimulator`. Basically, a user-friendly API allows you to create things in the simplest possible manner, preferably defaulting settings to sensible values if such sensible values exist. In case of thresholds, we should definitely default them to make the constructor easier to user :)